.github/workflows/release.yml

```diff
@@ -30,6 +30,7 @@ env:
   MSRV: "1.75"
   SCCACHE_CACHE_SIZE: "50G"
   BIN_NAMES: "iroh,iroh-relay,iroh-dns-server"
+  RELEASE_VERSION: ${{ github.event.inputs.release_version }}
```

Comment:
Given you have a whole script a little lower to set this in several ways I assume this is a leftover of an earlier version and should be removed?